boto/plugin.py

```diff
@@ -70,12 +70,8 @@ def _import_module(filename):
     (path, name) = os.path.split(filename)
     (name, ext) = os.path.splitext(name)
 
-    (file, filename, data) = imp.find_module(name, [path])
-    try:
-        return imp.load_module(name, file, filename, data)
-    finally:
-        if file:
-            file.close()
+    spec = importlib.machinery.PathFinder().find_spec(name, [path])
+    return importlib.util.module_from_spec(spec)S
```

Comment:
S in the end, typo? Not sure why this didn't throw a syntax error. Is this file even getting loaded?